src/preact/utils.js

```diff
@@ -53,8 +53,9 @@ export function useMergeRefs(refs) {
  * This passes through the value during development, because we render on Preact.
  * It's an annotation so that we can convert these values when we transform the
  * React build.
- * @param {string} name
- * @return {string}
+ * @param {T} name
```

Comment:
Can this really be anything?

Reply:
Good call. Constrained it to strings. The important aspect was that `T` is maintained in the output